spikeinterface/sortingcomponents/peak_localization.py

```diff
@@ -72,11 +103,25 @@ def _init_worker_localize_peaks(recording, peaks, method, nbefore, nafter, neigh
     worker_ctx['recording'] = recording
     worker_ctx['peaks'] = peaks
     worker_ctx['method'] = method
+    worker_ctx['method_kwargs'] = method_kwargs
     worker_ctx['nbefore'] = nbefore
     worker_ctx['nafter'] = nafter
-    worker_ctx['neighbours_mask'] = neighbours_mask
+    
     worker_ctx['contact_locations'] = contact_locations
-
+    worker_ctx['margin'] = margin
+    
+    
+    if method in ('center_of_mass', 'monopolar_triangulation'):
```

Comment:
I don't think this is needed because you already check the method before

Reply:
Lets leave this for when we will add other methods with other kwargs.